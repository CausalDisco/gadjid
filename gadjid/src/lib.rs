// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
