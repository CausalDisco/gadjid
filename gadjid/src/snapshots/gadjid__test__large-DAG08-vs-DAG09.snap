---
source: src/lib.rs
expression: "test(g_true, g_guess)"
---
g_true: 10008.DAG-100
g_guess: 10009.DAG-100
ancestor_aid:
  - 0.0014141414141414141
  - 14
oset_aid:
  - 0.01585858585858586
  - 157
parent_aid:
  - 0.1703030303030303
  - 1686
shd:
  - 0.007676767676767677
  - 38
t:
  - 2
  - 3
  - 4
  - 12
  - 13
  - 18
  - 19
  - 21
  - 23
  - 24
  - 26
  - 27
  - 32
  - 37
  - 39
  - 43
  - 45
  - 48
  - 49
  - 50
  - 51
  - 54
  - 55
  - 57
  - 59
  - 62
  - 63
  - 65
  - 67
  - 71
  - 72
  - 73
  - 79
  - 81
  - 82
  - 83
  - 89
  - 90
  - 91
  - 93
  - 94
  - 95
  - 96
  - 98
  - 99
y: 69
z:
  - 5
  - 7
  - 17
  - 29
  - 30
  - 44
  - 47
  - 52
  - 56
  - 60
  - 68
  - 74
  - 75
  - 77
  - 88
possible_descendants_of_t_in_g_guess:
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 12
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
  - 39
  - 40
  - 41
  - 42
  - 43
  - 44
  - 45
  - 46
  - 47
  - 48
  - 49
  - 50
  - 51
  - 52
  - 53
  - 54
  - 55
  - 56
  - 57
  - 58
  - 59
  - 60
  - 61
  - 62
  - 63
  - 64
  - 65
  - 66
  - 67
  - 68
  - 69
  - 70
  - 71
  - 72
  - 73
  - 74
  - 75
  - 76
  - 77
  - 78
  - 79
  - 80
  - 81
  - 82
  - 83
  - 84
  - 85
  - 86
  - 87
  - 88
  - 89
  - 90
  - 91
  - 92
  - 93
  - 94
  - 95
  - 96
  - 97
  - 98
  - 99
not_amenable_in_g_guess_wrt_t: []
proper_ancestors_of_y_in_g_guess_wrt_t:
  - 0
  - 1
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 14
  - 15
  - 16
  - 17
  - 20
  - 22
  - 25
  - 28
  - 29
  - 30
  - 31
  - 33
  - 34
  - 35
  - 36
  - 38
  - 40
  - 41
  - 42
  - 44
  - 46
  - 47
  - 52
  - 53
  - 56
  - 58
  - 60
  - 68
  - 69
oset_for_t_onto_y_in_g_guess:
  - 0
  - 1
  - 2
  - 4
not_validly_adjusted_for_in_g_guess_by_parents_of_t:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 12
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
  - 39
  - 40
  - 41
  - 42
  - 43
  - 44
  - 45
  - 46
  - 47
  - 48
  - 49
  - 50
  - 51
  - 52
  - 53
  - 54
  - 55
  - 56
  - 57
  - 58
  - 59
  - 60
  - 61
  - 62
  - 63
  - 64
  - 65
  - 66
  - 67
  - 68
  - 69
  - 70
  - 71
  - 72
  - 73
  - 74
  - 75
  - 76
  - 77
  - 78
  - 79
  - 80
  - 81
  - 82
  - 83
  - 84
  - 85
  - 86
  - 87
  - 88
  - 89
  - 90
  - 91
  - 92
  - 93
  - 95
  - 96
  - 97
not_validly_adjusted_for_in_g_guess_by_oset_for_t_onto_y:
  - 0
  - 1
  - 2
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 14
  - 15
  - 16
  - 17
  - 20
  - 22
  - 25
  - 28
  - 29
  - 30
  - 31
  - 33
  - 34
  - 35
  - 36
  - 38
  - 40
  - 41
  - 42
  - 44
  - 46
  - 47
  - 52
  - 53
  - 56
  - 58
  - 60
  - 61
  - 64
  - 66
  - 68
  - 69
  - 70
  - 74
  - 75
  - 76
  - 77
  - 78
  - 80
  - 84
  - 85
  - 86
  - 87
  - 88
  - 92
  - 97
not_validly_adjusted_for_in_g_guess_by_empty_set:
  - 0
  - 1
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 14
  - 15
  - 16
  - 17
  - 20
  - 22
  - 25
  - 28
  - 29
  - 30
  - 31
  - 33
  - 34
  - 35
  - 36
  - 38
  - 40
  - 41
  - 42
  - 44
  - 46
  - 47
  - 52
  - 53
  - 56
  - 58
  - 60
  - 61
  - 64
  - 66
  - 68
  - 69
  - 70
  - 74
  - 75
  - 76
  - 77
  - 78
  - 80
  - 84
  - 85
  - 86
  - 87
  - 88
  - 92
  - 97
not_validly_adjusted_for_in_g_guess_by_z:
  - 0
  - 1
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
  - 11
  - 14
  - 15
  - 16
  - 17
  - 20
  - 22
  - 25
  - 28
  - 29
  - 30
  - 31
  - 33
  - 34
  - 35
  - 36
  - 38
  - 40
  - 41
  - 42
  - 44
  - 46
  - 47
  - 52
  - 53
  - 56
  - 58
  - 60
  - 61
  - 64
  - 66
  - 68
  - 69
  - 70
  - 74
  - 75
  - 76
  - 77
  - 78
  - 80
  - 84
  - 85
  - 86
  - 87
  - 88
  - 92
  - 97
